rllib/agents/dqn/dqn.py

```diff
@@ -308,6 +312,25 @@ def update_target_if_needed(trainer, fetches):
         trainer.state["num_target_updates"] += 1
 
 
+def training_pipeline(workers, config):
+    local_replay_buffer = ReplayBuffer(config["buffer_size"])
+    rollouts = ParallelRollouts(workers)
+
+    # (1) Store experiences into the local replay buffer.
+    store_op = rollouts.for_each(StoreToReplayBuffer(local_replay_buffer))
+
+    # (2) Replay and train on selected experiences.
+    replay_op = LocalReplay(local_replay_buffer, config["train_batch_size"]) \
```

Comment:
It's not intuitively clear here, why we have to call `for_each` on the return value of LocalReplay (intuitively, I would think it's a single batch taken by the learner from the (only) replay_buffer). Could you add more comments here to explain?